sstables/sstables.cc

```diff
@@ -2343,7 +2344,7 @@ static future<bool> do_validate_compressed(input_stream<char>& stream, const sst
         auto expected_checksum = read_be<uint32_t>(buf.get() + compressed_len);
         auto actual_checksum = ChecksumType::checksum(buf.get(), compressed_len);
         if (actual_checksum != expected_checksum) {
-            sstlog.error("Compressed chunk checksum mismatch at offset {}, for chunk of size {}: expected={}, actual={}", offset, chunk_len, expected_checksum, actual_checksum);
+            sstlog.error("Compressed chunk checksum mismatch at chunk {}, offset {}, for chunk of size {}: expected={}, actual={}", i, offset, chunk_len, expected_checksum, actual_checksum);
```

Comment:
I think this wording is a bit confusing - is this an offset into the chunk or in the file? Why is the chunk number separate from it size? How about this:
```c++
sstlog.error("Compressed chunk checksum mismatch at offset {}, for chunk #{} of size {}: expected={}, actual={}", offset, i, chunk_len, expected_checksum, actual_checksum);
```

Reply:
Yes, this is better. I will send a follow-up.